config/locales/notices/en.yml

```diff
@@ -16,25 +16,24 @@ en:
       resend_email_success: We have resent your password reset email.
     password_changed: You changed your password.
     resend_confirmation_email:
-      success: We have resent your confirmation email.
+      success: We resent your confirmation email.
```

Comment:
might just be me, but I read this as "resent" as in, "I resent you" rather than "I resent the letter after it was returned to me"